docker/docker-compose.yml

```diff
@@ -0,0 +1,91 @@
+version: '3'
+volumes:
+  girder: {}
+  db: {}
+  worker: {}
+services:
+  docker:
+    image: 'docker:dind'
```

Comment:
@manthey is the resident expert on docker in/alongside docker, but [dind is considered obsolete](https://github.com/jpetazzo/dind#another-word-of-warning) as far as I know.